met_connectors/src/frost/fetch.rs

```diff
@@ -143,19 +143,22 @@ fn json_to_data_cache(
                 curr_obs_time = curr_obs_time + period;
             }
 
-            Ok(((station_id, data), location))
+            Ok((Timeseries{tag:station_id, values:data}, location))
```

Comment:
Since we don't use `location` in this closure (will we use it in the future?), is there a smarter way to decouple it from the timeseries data? Maybe we can `unzip` in `extract_data`, but it's a bit annoying to get it right with all the Results and Options